lib/flight.js

```diff
@@ -124,3 +124,13 @@ Flight.prototype.rightFlip = function() {
 Flight.prototype.wave = function() {
   return this.connection.animate("wave", 750);
 };
+
+/**
+ * Tells the drone to perform horizontal trimming.
+ *
+ * @return {null}
+ * @publish
+ */
+Flight.prototype.ftrim = function() {
```

Comment:
Not sure this is actually necessary? Is it sufficient to add the command to commands.js?